tests/mock_network/node_ageing.rs

```diff
@@ -116,19 +132,17 @@ fn relocate_causing_split() {
 #[ignore]
```

Comment:
Should the new id of the joining node be in the sub-prefix that contain the relocation destination (i.e either a prefix with one more bit that the one we knew, or even a few more bits (maybe 3)).

Reply:
Interesting idea. I guess there is always some probability that the section splits multiple times by the time the relocated node joins, but that probability rapidly decreases with each bit added, so this seem like a decent fix. I would address this in a separate PR though, if you don't mind.

Reply:
There was already an issue for this: https://github.com/maidsafe/routing/issues/1861 - I added your idea there.